sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/FunctionRegistry.scala

```diff
@@ -302,14 +302,14 @@ object FunctionRegistry {
     expression[Skewness]("skewness"),
     expression[ApproximatePercentile]("percentile_approx"),
     expression[ApproximatePercentile]("approx_percentile"),
-    expression[StddevSamp]("std"),
-    expression[StddevSamp]("stddev"),
+    expressionWithAlias[StddevSamp]("std"),
+    expressionWithAlias[StddevSamp]("stddev"),
     expression[StddevPop]("stddev_pop"),
-    expression[StddevSamp]("stddev_samp"),
+    expressionWithAlias[StddevSamp]("stddev_samp"),
```

Comment:
can we put them closer if they use the same expression?

Reply:
Sure